src/main/java/org/quiltmc/loader/impl/metadata/qmj/V1ModMetadataReader.java

```diff
@@ -309,6 +310,10 @@ private V1ModMetadataImpl readFields(JsonLoaderValue.ObjectImpl root) {
 				throw new ParseException("Oh no! This version of Quilt Loader doesn't support hashed mappings, please update Quilt Loader to use this mod.");
 			}
 
+			if (mappings.equals("mojang") && !QuiltLoader.getMappingResolver().getNamespaces().contains("mojang")) {
+				throw new ParseException("Oh no! This version of Quilt Loader doesn't support Mojang mappings, please update Quilt Loader to use this mod.");
```

Comment:
This copy-pastey comment doesn't make sense here compared to the Hashed Mojmap one; how about:
```suggestion
				throw new ParseException("Oh no! This Quilt Loader installation isn't configured to support Mojang mappings.");
```
(potential remedies to this are to be determined once uhhhh, we figure out a good way to provide mojmap)